packages/core/strapi/lib/commands/__tests__/data-transfer/transfer.test.js

```diff
@@ -139,6 +156,8 @@ describe('Transfer', () => {
     });
   });
 
+  it.todo('uses local Strapi destination when to is not specified');
```

Comment:
Can't be this test be implemented?

Reply:
Not until pull is merged :)

We'll update this after your PR is merged. Or add the test to yours directly.